SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -420,6 +427,9 @@ void UGlobalStateManager::SetAcceptingPlayers(bool bInAcceptingPlayers)
 	// Set the AcceptingPlayers state on the GSM
 	Schema_AddBool(UpdateObject, SpatialConstants::DEPLOYMENT_MAP_ACCEPTING_PLAYERS_ID, static_cast<uint8_t>(bInAcceptingPlayers));
 
+	// Set the schema hash on the GSM
+	Schema_AddUint32(UpdateObject, SpatialConstants::DEPLOYMENT_MAP_SCHEMA_HASH, NetDriver->ClassInfoManager->SchemaDatabase->SchemaDescriptorHash);
```

Comment:
Wondering if we should change `SetAcceptingPlayers` name should be changed since this is new doing a few things

Reply:
Changed to SetupAcceptingPlayerState, what do you think?

Reply:
Dislike, this is still a function which can be used more than once so the word Setup doesn't make sense.
Perhaps we should break this into multiple functions such as `SetAcceptingPlayers` `SetDeploymentMapURL` `SetSchemaHash`.
We're doing them all in one here and it doesn't make sense to do all of these things when you for example `SetAcceptingPlayers(false)`

Reply:
Yeah I agree, I have moved it to SetCanBeginPlay which seems more appropriate as this isn't going to change during the deployment